REFERENCE.md

```diff
@@ -397,6 +397,16 @@ Currently only implemented for systemd based service.
 
 Default value: ``undef``
 
+##### `timeout_stop`
+
+Data type: `Optional[String]`
```

Comment:
i feel like we can do better than this
but that better type would either belong here:
https://github.com/camptocamp/puppet-systemd/tree/master/types

unless a general one already exists here: https://github.com/puppetlabs/puppetlabs-stdlib/tree/main/types

Reply:
I totally agree that "String" is very ... liberal - and I checked both modules before.

Let's see if there's nice regex to be drafted in a puppet type before merging.